### PR TITLE
fix: error message for --version

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix, refactor

**Did you add tests for your changes?**
hard to test as repo has all packages present, Manual testing.
**If relevant, did you update the documentation?**

**Summary**
better error message. `External package` is more general hence use the specific package name.

Screenshot at 2020-12-07 07-27-06

 

**Does this PR introduce a breaking change?**
NO

**Other information**
https://github.com/webpack/webpack.js.org/pull/4261#discussion_r537054656

### CODE REVIEW

**Review by alexander-akait — COMMENTED** [review]
Do we have tests for this?

**Comment by snitin315** [comment]
No probably because the repo already contains all these packages.

File: packages/webpack-cli/lib/groups/runVersion.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const outputVersion = (args) => {
             const { name, version } = require(`${usedCommands[0].packageName}/package.json`);
             logger.raw(`${name} ${version}`);
         } catch (e) {
-            logger.error('Error: External package not found.');
+            logger.error(`Error: package '${usedCommands[0].packageName}' not found.`);
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/groups/runVersion.js:56]: codecove says what we don't have tests on this, let's add this

Comment (snitin315) [reply packages/webpack-cli/lib/groups/runVersion.js:56]: Any suggestions on how to test it?

Comment (alexander-akait) [reply packages/webpack-cli/lib/groups/runVersion.js:56]: mocking

**Comment by alexander-akait** [comment]
Fixed in master, anyway thanks for the PR

**Pull request closed without merging**